text/0092-logs-vision.md

```diff
@@ -0,0 +1,128 @@
+
+Existing log formats can be unambiguously mapped to this data model. Reverse 
+mapping from this data model is also possible to the extent that the target log 
+format has equivalent capabilities.
+
+We will produce mapping recommendations for commonly used log formats.
+
+### Log Protocol
+
+Armed with the Log Data model we will aim to design a high performance protocol
+for logs, which will pursue the same [design goals](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/proto-design-goals.md)
+as we had for traces and metrics protocol design.
+
+Most notably the protocol will aim to be highly reliable, have low resource
+consumption, be suitable for all participant nodes, ensure high throughput,
+allow backpressure signalling and be load-balancer friendly.
+
+The reason for this design is to have a single protocol that can deliver
+logs, traces and metrics via one connection and satisfy all design goals
+(as far as we know this does not exist today).
+
+### Unified Collection
+
+We aim to support high-performance, unified Agent and Collector that supports
+logs, traces and metrics in one package, symmetrically and uniformly for all 3
+types of telemetry data (see also [Collector vision](https://github.com/open-telemetry/opentelemetry-collector/blob/8310e665ec1babfd56ca5b1cfec91c1f997f4f2c/docs/vision.md)).
+
+The unified Collector will support multiple log protocols including the newly
+designed OpenTelemetry log protocol.
+
+Unified collection is important for the following reasons:
+- One agent (or one collector) to deploy and manage.
+- One place of configuration for target endpoints, authentication tokens, etc.
+- Uniform tagging of all 3 types of telemetry data (enrichment by attributes
+  of resources where the data comes from or by user-defined attributes),
+  enabling correct correlation across Resource dimensions later on the backend.
+
+## Cloud Native
+
+We will have best-in-class support for logs emitted in cloud native environments
+(e.g. Kubernetes, serverless, etc), including legacy applications running
+in such environments. This is inline with our CNCF mission.
+
+## Support Legacy
+
+We will produce guidelines on how legacy applications can emit logs in a
+manner that makes them compatible with OpenTelemetry's approach and enables
+telemetry data correlation. We will also have a reasonable story around
+logs that are emitted by sources over which we may have no control and which
+emit logs in pre-defined formats via pre-defined mediums (e.g. flat file logs,
+Syslog, etc).
+
+We will have technical solutions or guidelines for using popular logging
+libraries in OpenTelemetry-compatible manner and we may produce logging
+libraries for languages where gaps exist.
+
+This is important because we believe software that was created before
+OpenTelemetry should not be disregarded and should benefit from OpenTelemetry
+efforts where possible.
+
+### Auto-instrumenting
```

Comment:
minor: consider `Auto-instrumentation`

Other places in OpenTelemetry use **auto-instrumentation** instead of **auto-instrumenting**, for example:
https://github.com/open-telemetry/opentelemetry-auto-instr-java

Reply:
Fixed.